### DISCUSSION TITLE
A few UI issues

### DISCUSSION THREAD

**Discussion by YaakovDavis** [Lightning-AI/dl-fundamentals#36] [General discussions]
1. Code segments don't show commas, dots, colons, and parentheses (a screenshot is attached).
2. In multi-video units, the website doesn't remember which ones I watched.
3. When switching to another video while a video is playing, playback doesn't stop.

![2023-04-08 11_51_24-4 3 Training a Multilayer Neural Network in PyTorch (PART 1-5) - Lightning AI](https://user-images.githubusercontent.com/965750/230703786-86e1b343-8e37-476a-b2d7-5d1f9be5b749.png)

**Comment by rasbt** [comment]
Sorry, missed this last week somehow! Thanks a lot for the note, I will discuss this with out web dev, and hopefully we can address all of these asap.

**Comment by rasbt** [comment]
Happy to report that the

> When switching to another video while a video is playing, playback doesn't stop.

issue is fixed now!
